hack/run-image-scan.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+success_images=()
+failed_images=()
+
+if [ "$#" -lt 1 ] || [ "${1}" == "--help" ]; then
+  cat << EOF
+Usage:
+  $(basename $0) -i image1,image2
+  $(basename $0) -f fs1,fs2
+Example:
+  $(basename $0) -i quay.io/kubermatic/kubermatic:v2.21.1,quay.io/kubermatic/kubermatic:v2.21.0
+  $(basename $0) -f https://github.com/kubermatic/kubeone
+EOF
+  exit 0
+fi
+
+function image_scan() {
+  for i in "${image_array[@]}"; do
+    EXIT_CODE=0
+    echo -e "Starting Vurnability scanning for ${i}"
+    trivy image --exit-code 1 --no-progress ${i} || EXIT_CODE=1
+    if [[ EXIT_CODE -eq 1 ]]; then
+      failed_images[${#failed_images[@]}]=${i}
+    else
+      success_images[${#failed_images[@]}]=${i}
+    fi
+  done
+  if [ "${#failed_images[@]}" -gt 0 ]; then
+    echo -e "Vurnabilities found on the images ${failed_images[@]}"
+  else
+    echo -e "No vurnabilitiies found on the images ${success_images[@]}"
```

Comment:
Let's use `echodate` here. `echodate` is coming from our bash library, so you need to source that library at the beginning of your script: https://github.com/kubermatic/kubermatic/blob/2f8eff72af97022d2f446c8950677f799fcfd437/hack/gen-api-client.sh#L20-L23

Reply:
Done